runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -842,10 +910,16 @@ gcParseSovereignArguments(J9JavaVM *vm)
 	IDATA index = -1;
 	PORT_ACCESS_FROM_JAVAVM(vm);
 
-	if (!gcParseXlpOption(vm)) {
+	IDATA xlpOptionIndex = gcParseXlpOption(vm);
+	if (xlpOptionIndex == -1) {
```

Comment:
should be `(-1 == xlpOptionIndex)`

Reply:
Thanks for catching that. 

See https://github.com/eclipse/openj9/pull/7476/commits/ba21f7cb46f8f75d4c79e77e0a8eb6447b13f149